python/dstat-kudu/kudu_dstat.py

```diff
@@ -4,16 +4,12 @@
 import sys
 import tempfile
 import time
+from kudu.client import Partitioning
```

Comment:
As discussed, I'll update the documentation to reflect that users need to install the python client through pip.  I was thinking i'd do that as a separate PR